src/xenia/kernel/xboxkrnl/xboxkrnl_rtl.cc

```diff
@@ -25,8 +25,10 @@
 #include "xenia/kernel/xevent.h"
 #include "xenia/kernel/xthread.h"
 
-// For FileTimeToSystemTime and SystemTimeToFileTime:
+#if XE_PLATFORM_WIN32
```

Comment:
If you mean the platform check, yes it is needed. Windows implements timegm but calls it _mkgmtime, so the #define is used for that.